spec/requests/providers/vehicles/ages_spec.rb

```diff
@@ -92,7 +92,7 @@
       it 'displays error' do
         subject
         expect(response.body).to include('govuk-error-summary')
-        expect(response.body).to include('Select Yes if your client bought the car over three years ago')
+        expect(response.body).to include('Select yes if your client bought the vehicle over 3 years ago')
```

Comment:
If we expect the translation rather than the actual text it could eliminate the need for amending the tests when content is updated.

Reply:
Fair point. I've pushed an extra commit that fixes this in a few places (all that I could find, although there might be some that I've missed) if you fancy giving it a quick re-review.